extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -1,19 +1,25 @@
 <?php //phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
 
 /**
- * Tiled Gallery block. Depends on the Photon module.
+ * Tiled Gallery block.
+ * Relies on Photon, but can be used even when the module is not active.
```

Comment:
I took the opportunity to update this since it wasn't valid anymore since #13471.